src/shellout.rs

```diff
@@ -24,7 +24,9 @@ fn exec_lpass_input(args: &[&str], data: &str) -> Result<String, LPassError> {
     }
     let mut child = child.unwrap();
 
-    child.stdin.as_mut().unwrap().write_all(data.as_bytes());
+    if let Err(err) = child.stdin.as_mut().unwrap().write_all(data.as_bytes()) {
+        return Err(LPassError::new("Could write data", Some(Box::from(err))));
```

Comment:
Should the string be _Could not write data_ here?

Reply:
You are correct - I guess I was a little quick to correct here when I got compiler warning.